packages/runtime/container-runtime/src/gc/gcEarlyAdoption.md

```diff
@@ -0,0 +1,130 @@
+
+```ts
+"Fluid.GarbageCollection.ThrowOnTombstoneLoad": true
+```
+
+Now, even with `Fluid.GarbageCollection.ThrowOnTombstoneLoad` set to true, changes to a Tombstoned object will be allowed.
+This is required for the advanced recovery options to work, explained [below](#advanced-back-door-recovering-and-reviving-tombstoned-objects).
+
+To instruct FF to treat Tombstoned objects as if they are truly not present in the document, use this Config Setting:
+
+```ts
+"Fluid.GarbageCollection.ThrowOnTombstoneUsage": true
+```
+
+### In case of emergency: Setting the gcTombstoneGeneration
+
+GC includes a mechanism for Tombstone by which all new documents may be stamped with a "Generation" number,
+and if set then Tombstone is only enforceable for documents of the latest Generation. This number is specified
+via the `gcTombstoneGeneration` GC Option, and will not change over the lifetime of a given document.
+
+In case a bug is released that is found to cause GC errors, a bump to the gcTombstoneGeneration can be incuded
+with the fix, which will prevent any user pain for those potentially affected documents that were exposed to the bug.
+
+If gcTombstoneGeneration is unset, Tombstone enforcement will be enabled/disabled as otherwise configured.
+In other words, until you start using this, Tombstone enforcement will apply to all documents.
+
+### Advanced "Back door": Recovering and reviving Tombstoned objects
+
+If your application has Tombstone enabled and your users are encountering Tombstones - even at the point where
+Tombstone enforcement is enabled - there is a way to still access these objects to recover them and property
+reference them ("revival"). However, please understand that this is an advanced and unsupported path that may
+be immediately deprecated at any time.
+
+As mentioned above, bumping the gcTombstoneGeneration will free up impacted documents, but that's a permanent
+mitigation - those documents will never be exposed to GC Tombstone or Sweep.
+
+If there's a particular codepath in your application where objects being loaded may be Tombstoned,
+you may use this advanced "back door" to recover them and then properly reference them, thus restoring the document.
+For this to work, you must disable the `Fluid.GarbageCollection.ThrowOnTombstoneUsage` Config Setting.
+
+When a Tombstoned object (via `handle.get()`) fails to load, the 404 response error object has an `underlyingResponseHeaders` with the
+`isTombstoned` flag set to true: i.e. `error.underlyingResponseHeaders?.isTombstoned === true`. In this case,
+you may turn around and use `IContainerRuntime.resolveHandle` with `allowTombstone: true` in `IRequest.headers` to request
+the object again - this time it will succeed.
+
+To be very clear once again - This path uses deprecated APIs (`resolveHandle`) and comes with no guarantees of support.
+
+### Tombstones and the Summarizer Client
+
+Note: The Summarizer client will _never_ throw on usage or load of a Tombstoned object.
```

Comment:
Should we mention why?

Reply:
I tried briefly but wasn't sure how to say it succinctly.  Make a suggestion comment if you have an idea what to say.